extension/azure-devops-extension.json

```diff
@@ -2,7 +2,7 @@
   "manifestVersion": 1,
   "id": "ado-aws-widgets-dev",
   "publisher": "baileyeverts2",
-  "version": "0.0.45",
+  "version": "0.0.127",
```

Comment:
:eyes: that's quite a bump!

Reply:
you can really see how many tries it took lol

Reply:
:laughing: been there before, many times